Notifier-service/src/server.ts

```diff
@@ -16,9 +18,14 @@ dotenv.config()
 */
 
 
-let app = new App();
-app.init()
-app.start((port) => {
-    console.log("Listening on port :"+port);
-})
+// Ensure required SERVER_PORT PORT have datatype of number
+if(isNaN(Number(process.env.SERVER_PORT))) {
+    throw new Error('The SERVER_PORT must be a number!')
```

Comment:
* If throwing error then using if-else is not required.
* The exception will take case of safe guarding the code for app.

Reply:
So, should i remove the if, else condition and only leave the line "isNaN(Number(process.env.SERVER_PORT)" ?